PyRDF/Node.py

```diff
@@ -86,6 +87,12 @@ def __setstate__(self, state):
         Retrieves the state dictionary of the current
         node and sets the instance variables.
 
+        Parameters
+        ----------
+        state : Python Dictionary
```

Comment:
just dictionary?  Since we are in Python...

Reply:
although if you change it, all the other docs should be consistent (also if it is a list)